### PR TITLE
V01 trident skirt remix

### PR DESCRIPTION
**Pull request by damian-gdes**
* [:heavy_check_mark: ] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [ :heavy_check_mark:] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [ :heavy_check_mark:] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [:heavy_check_mark: ] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [:heavy_check_mark: ] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 

Designed, printed and assembled by myself and few other ppl - photos included

  * [:heavy_check_mark: ] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [:heavy_check_mark: ] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [:heavy_check_mark: ] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [:heavy_check_mark: ] Submitted STL files are oriented and scaled properly for printing.
  * [:heavy_check_mark: ] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?

V0.1 Extended tophat hinge & V0.1 Trident skirt remix with 12864 LCD

#### How was it tested? 

Printed and assemblied myself and few other people, some photos included in mod/images folders

#### Any background context you want to provide?

Hinged version was designed because more then half my prints are made of impact pla and also many filaments changes daily basis so openable top was necessity, skirt is purely cosmetics as i didnt want use v0 lcd and had 12864 one laying :)

#### Screenshots (if appropriate)

Included in mod/images folders

#### Further notes

Thanks for hartk and daver for help

### CODE REVIEW

File: printer_mods/README.md [hunk 1]
```diff
@@ -122,6 +122,8 @@ like so:
 | GiulianoM | [Fly F407ZG Mount](./GiulianoM/FlyF407ZG) | Mount for the Fly F407ZG: https://github.com/FLYmaker/FLYF407ZG | :x: | :x: | :heavy_check_mark: | :x: |
 | Gizzle | [Ender 3 and Ender 3 Pro Switchwire Mod](./Gizzle/ender-3_(pro)_switchwire)| Mod for the Ender 3 and Ender 3 Pro, which adapts the kinematics and most other parts of the Voron Switchwire. | :x: | :x: | :x: | :heavy_check_mark: |
 | GSL12 | [Pocketwatch Nema 17](./GSL12/pocketwatch_nema17)| Extruder like Pocketwatch, but with Nema17 instead of Nema14. Maybe for V0 fork with 2020 extrusions, Nema17 and MGN9 or other printers. | :grey_question: | :grey_question: | :grey_question: |:x:|
+| Golas | [V0.1 Extended tophat hinge](./golas/v0-extended-tophat-hinge)| Extended tophat for v0.1 with hinges (integrated or screwed ones). | :heavy_check_mark: | :x: | :x: |:x:|
```
Comment (cbc02009) [inline printer_mods/README.md:125]: please add an :x: for the trident column for both these mods.

**Review by cbc02009 — CHANGES_REQUESTED** [review]
There are so many files in this mod that I'm hesitant to approve it without some sort of manual or installation instructions.

**Comment by damian-gdes** [comment]
> There are so many files in this mod that I'm hesitant to approve it without some sort of manual or installation instructions.

For skirt mod there is manual in pdf, for tophat i am making one, clarification - those are two mods at one PR - skirt and tophat.

**Comment by cbc02009** [comment]
Yup, I saw that. If you want the smaller mod to be merged in first you can create a separate pull request for it. Otherwise just post a comment when the manual is done and I'll review it.

**Comment by damian-gdes** [comment]
> Yup, I saw that. If you want the smaller mod to be merged in first you can create a separate pull request for it. Otherwise just post a comment when the manual is done and I'll review it.

No problem, I'll finish manual shortly and upload for repo, thx for reply.

File: printer_mods/README.md (hunk 1 above)
Comment (damian-gdes) [reply printer_mods/README.md]: Added

**Comment by damian-gdes** [comment]
> Yup, I saw that. If you want the smaller mod to be merged in first you can create a separate pull request for it. Otherwise just post a comment when the manual is done and I'll review it.

Simple manual is in repo now :)

**Comment by damian-gdes** [comment]
Manual was added as requested

**Comment by damian-gdes** [comment]
> There are so many files in this mod that I'm hesitant to approve it without some sort of manual or installation instructions.

Manual added

**Comment by cbc02009** [comment]
I saw, there's just a lot of files so I haven't had a chance to go over the whole thing yet.

**Comment by damian-gdes** [comment]
> I saw, there's just a lot of files so I haven't had a chance to go over the whole thing yet.

Ahh ok mate didn't know what comment and where is proper :) CR Readme? Readme was edited too like you wanted - i assume now i wait for stl checks :)

**Comment by cbc02009** [comment]
Right, but I haven't reviewed it make sure the changes are correct, so the tag stays on until one of us reviews them.

**Comment by damian-gdes** [comment]
Any chance for review?

**Review by cbc02009 — APPROVED** [review]

**Pull request merged**